website/content/en/docs/contribution-guidelines/issue-lifecycle.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Issue Lifecycle
+weight: 30
+---
+
+The Operator SDK project tracks bugs, feature requests, and questions
+with [GitHub
+issues](https://github.com/operator-framework/operator-sdk/issues).
```

Comment:
You can keep your lines short by putting the urls at the bottom of the document. 
```
[GitHub issues][gh-issues]

...
[gh-issues](https://github.com/operator-framework/operator-sdk/issues)
```
We've done that in a number of places.